lab5/solid_principle.py

```diff
@@ -0,0 +1,142 @@
+from math import pi
+
+# --------------------------------------------------------------------------------------------
+# SRP - Single Responsibility Principle
+class Customer:
+    """
+    Create a customer.
+    """
+    def __init__(self, name, mobile):
+        self.name = name
+        self.mobile = mobile
+
+class GetCustomer(Customer):
```

Comment:
this inheritance is not logical, all getters and setters should be inside the class that defines the properties. Take a look at @property decorator